### DISCUSSION TITLE
llama.cpp is missing support for special tokens for tokenizing bos/eos

### DISCUSSION THREAD

**Discussion by ycros** [viniciusarruda/llama-cpp-chat-completion-wrapper#1] [General]
Because it doesn't have a special_tokens_map.json type configuration like on a typical hf transformers model.

This PR against llama.cpp would fix it and has more details and discussion around the issue.

https://github.com/ggerganov/llama.cpp/pull/1931

**Comment by viniciusarruda** [comment]
I think the issue is even broader. I've created a discussion [here](https://github.com/viniciusarruda/llama-cpp-chat-completion-wrapper/discussions/2).
